tools/build_steps.sh

```diff
@@ -19,12 +36,13 @@ function before_build {
             sudo chmod 777 /usr/local/include
             touch /usr/local/include/.dir_exists
         fi
-        source ${ROOT_DIR}/multibuild/osx_utils.sh
-        get_macpython_environment ${MB_PYTHON_VERSION} venv
+        # get_macpython_environment ${MB_PYTHON_VERSION} venv
+        python3.9 -m venv venv
+        source venv/bin/activate
         # Since install_fortran uses `uname -a` to determine arch,
         # force the architecture
         arch -${PLAT} bash -s << EOF
-source ${ROOT_DIR}/gfortran-install/gfortran_utils.sh
+source tools/gfortran_utils.sh
```

Comment:
In #216 I replaced this with an alias (and moved it inside the `if $IS_OX`), since gfortran is installed in the macos image.
```
alias gfortran=gfortran-15
```